docs/3.0.0-beta.x/content-api/api-endpoints.md

```diff
@@ -2,32 +2,62 @@
 
 When you create a `Content Type` you will have a certain number of **REST API endpoints** available to interact with it.
 
-As an **example** let's consider the `Restaurant` as a **Content Type** and `Openning_hours` as a **Group** for the next steps.
+As an **example** let's consider the following models:
+
+**Content Type**:
+
+- `Restaurant`
+
+**Components**:
+
+- `Openning hours` (category: `restaurant`)
+- `Title With Subtitle` (category: `content`)
+- `Image With Description` (category: `content`)
 
 :::: tabs
 
-::: tab "Content Type"
+::: tab "Content Types"
 
 ### `Restaurant` Content Type
 
-| Fields        | Type   | Description                      | Options      |
-| :------------ | :----- | :------------------------------- | :----------- |
-| name          | string | Restaurant's title               |              |
-| cover         | media  | Restaurant's cover image         |              |
-| opening_hours | group  | Restaurant's opening hours group | `repeatable` |
+| Fields        | Type        | Description                          | Options      |
+| :------------ | :---------- | :----------------------------------- | :----------- |
+| name          | string      | Restaurant's title                   |              |
+| cover         | media       | Restaurant's cover image             |              |
+| content       | dynamiczone | The restaurant profile content       |              |
+| opening_hours | component   | Restaurant's opening hours component | `repeatable` |
 
 :::
 
-::: tab Group
+::: tab Components
 
-### `Opening_hours` Group
+### `Opening hours` Component
 
 | Fields       | Type   | Description         |
 | :----------- | :----- | :------------------ |
 | day_interval | string | Meta's day interval |
 | opening_hour | string | Meta's opening hour |
 | closing_hour | string | Meta's closing hour |
 
+---
```

Comment:
I don't understand what you tried to do here.
For the following lines. Two times the same compo but not exactly the same.